go/project_structure/operator.md

```diff
@@ -0,0 +1,59 @@
+# Operator
+
+This page describes common code structure for our operators.
+
+The example below is taken from [aws-operator] on 2019/11/01. Most of the files
+and directories are elided for simplification. Note that not all packages are
+described as this page should be extended over time.
+
+```
+./service/
+├── collector/
+├── controller/
+│   ├── controllercontext/
+│   ├── internal/
+│   │   ├── adapter/
+│   │   ├── changedetection/
+│   │   ├── cloudconfig/
+│   │   ├── cloudformation/
+│   │   ├── ebs/
+│   │   ├── encrypter/
+│   │   ├── templates/
+│   │   └── unittest/
+│   ├── key/
+│   └── resource/
+│   │   ├── accountid/
+│   │   ├── ...
+│   │   └── tcnpoutputs/
+│   ├── cluster.go
+│   └── machine_deployment.go
+├── internal/
+│   ├── accountid/
+│   ├── credential/
+│   └── locker/
+└── service.go
+```
+
+- `service/collector` package contains code for Prometheus collectors.
```

Comment:
It would be good to extend that part a little. I'm not an expert here. Maybe some add some file examples like `collector.go`, `asg.go`, `set.go` etc.